src/styles/block/card___block.scss

```
@@ -0,0 +1,128 @@
+@import "../utils/variables";
+@import "../utils/mixin";
+@import "../utils/reset";
+
+.cards {
+  font-family: inherit;
+  &__block {
+    display: flex;
+    flex-direction: column;
+    border: $border-width solid #f3f3f3;
+    border-radius: 5px;
+    background: $color-white;
+    width: $card-width;
+    margin-bottom: 46px;
+  }
+
+  &__block:nth-child(4n) {
+    margin-right: 0;
+  }
+
+  &__block:nth-last-child(-n + 4) {
+    margin-bottom: 0;
+  }
+    
+  &__img {
+    display: flex;
+    width: 160px;
+    height: 134px;
+    padding: 32px 19px 40px;
+  }
+
+  &__name-item {
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+    padding: 0 16px 4px;
+    color: $color-first;
+  }
+
+  &__name-article {
+    color: $color-second;
+    font-size: 10px;
+    line-height: 14px;
+    font-weight: 400;
+    text-align: left;
+    padding: 0 16px 16px;
+  }
+
+  &__card-rate {
+    display: flex;
+    padding-left: 16px;
+    padding-right: 16px;
+    justify-content: space-between;
+    align-items: flex-end;
+  }
+
+  &__stars {
+    display: flex;
+  }
+
+  .stars-star {
+    background-image: url(/images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+    @include icon ($size);
+    padding-left: 4px;
+  }
+
+  &__stars :not(:last-child) {
+    background-image: url(/images/star-active.svg);
+  }
```

```suggestion
  .stars-star {
    background-image: url(./images/star.svg);
    background-repeat: no-repeat;
    background-position: center;
    @include icon ($size);
    padding-left: 4px;
  }

  &__stars :not(:last-child) {
    background-image: URL(./images/star-active.svg);
  }
```

If you initial address for img with / (slash), browser start searching file with root folder. Sometimes it can be reason for error, use dots to set the address for files

---

Before committing my homework, I had tried all variances of the url with the dot, without or with double dots it didn't work at all. Only this way url (/img... matched to my case.
Why caps lock url is important?